pkgs/development/python-modules/colcon-cmake/default.nix

```
@@ -0,0 +1,25 @@
+{ lib, fetchgit, buildPythonPackage, colcon-core, colcon-library-path, colcon-test-result, pytest }:
+
+buildPythonPackage rec {
+  pname = "colcon-cmake";
+  version = "0.2.26";
+  src = fetchgit {
+    url = "https://github.com/colcon/colcon-cmake.git";
+    rev = "refs/tags/${version}";
+    deepClone = true;
+    sha256 = "H8jKNTXDd3HW8FLejgUlmhW1DQXXrZ/HOcHTrhol3ms=";
+  };
```

Gotten from `nix run nixpkgs#nix-prefetch-github -- --nix colcon colcon-cmake --rev 0.2.26 --no-fetch-submodules`
```suggestion
  src = fetchFromGitHub {
    owner = "colcon";
    repo = "colcon-cmake";
    rev = version;
    sha256 = "wL00sem/g4iLIuIX5nGlxUjVfkGQWAprw5iKrIVRXIg=";
  }
```